jsk_panda_robot/panda_eus/models/panda.yaml

```diff
@@ -0,0 +1,19 @@
+larm:
+  - panda_joint1 : larm-collar-y
+  - panda_joint2 : larm-shoulder-p
+  - panda_joint3 : larm-shoulder-y
+  - panda_joint4 : larm-elbow-p
+  - panda_joint5 : larm-wrist-r
+  - panda_joint6 : larm-wrist-p
+  - panda_joint7 : larm-wrist-y
+
+larm-end-coords:
+  parent: panda_link8
+  translate: [0, 0, 0.1]
+  rotate : [-0.35740972270209, -0.86285515644477, -0.35740630816298, 98.42112728719206]
+
+angle-vector:
+  reset-pose: [ 0.0, -45.0, 0.0, -135.0, 0.0, 90.0, 45.0,
+                0.0, -75.0 ]
+  reset-manip-pose: [ 0.0, -45.0, 0.0, -135.0, 0.0, 90.0, 45.0,
+                      0.0, -75.0 ]
```

Comment:
`reset-pose` seems different from https://github.com/jsk-ros-pkg/jsk_robot/blob/93919243019ae06be43874022062abbeee00349e/jsk_panda_robot/panda_eus/models/dual_panda.yaml#L32-L34
Also, `reset-pose` and `reset-manip-pose` are the same.
Do you have any reason why you choose this angle-vector as `reset-pose` and `reset-manip-pose`?

Reply:
In https://github.com/yuki-asano/jsk_robot/pull/1, I added notice about the difference of `reset-pose` and `reset-manip-pose` (and also `end-coords`) to README

Reply:
After https://github.com/yuki-asano/jsk_robot/pull/3, `pannda` and `dual_panda` use the same `end-coords`.